src/usdb_syncer/gui/song_table/song_table.py

```diff
@@ -149,20 +149,23 @@ def _setup_view(self) -> None:
                 existing_state = True
                 self._search.order = Column(header.sortIndicatorSection()).song_order()
                 self._search.descending = bool(header.sortIndicatorOrder().value)
+
+        header.setSectionsMovable(True)
+        header.setSectionsClickable(True)
+        header.setStretchLastSection(False)
+        header.setMinimumSectionSize(0)
```

Comment:
Maybe set the minimum to what was formerly the fixed size of the narrowest columns?